airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -139,7 +139,7 @@ public AirbyteCatalog discover(JsonNode config) throws Exception {
               Optional.ofNullable(config.get("database")).map(JsonNode::asText),
               Optional.ofNullable(config.get("schema")).map(JsonNode::asText))
                   .stream()
-                  .map(t -> CatalogHelpers.createAirbyteStream(t.getName(), t.getFields())
+                  .map(t -> CatalogHelpers.createAirbyteStream(t.getName(), t.getSchemaName(), t.getFields())
```

Comment:
mysql doesn't have schemas. what will happen in this case? null? or will it use the db name?

Reply:
Good question. 

This is handled here: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java#L448

I've confirmed it by printing out the returned catalog in the test:
```
io.airbyte.protocol.models.AirbyteCatalog@4f4c88f9[streams=[io.airbyte.protocol.models.AirbyteStream@6a756082[name=db_oKRDqDpNNt.id_and_name_without_pk,jsonSchema={"type":"object","properties":{"updated_at":{"type":"string"},"name":{"type":"string"},"id":{"type":"number"}}},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[],namespace=db_oKRDqDpNNt,additionalProperties={}], io.airbyte.protocol.models.AirbyteStream@59c70ceb[name=db_oKRDqDpNNt.id_and_name,jsonSchema={"type":"object","properties":{"updated_at":{"type":"string"},"name":{"type":"string"},"id":{"type":"number"}}},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[[id]],namespace=db_oKRDqDpNNt,additionalProperties={}], io.airbyte.protocol.models.AirbyteStream@46f902e0[name=db_oKRDqDpNNt.full_name_composite_pk,jsonSchema={"type":"object","properties":{"updated_at":{"type":"string"},"last_name":{"type":"string"},"first_name":{"type":"string"}}},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[[first_name], [last_name]],namespace=db_oKRDqDpNNt,additionalProperties={}]],additionalProperties={}]
```

the `namespace` field is filled, and matches the stream name, so think we are good